packages/core/src/types/const/const.ts

```diff
@@ -163,3 +163,5 @@ export const DEFAULT_SLIDE = {
         height: 300,
     },
 };
+
+export const UNIVER_INTERNAL = '__univer_internal__';
```

Comment:
Document required.

Reply:
When you modify the kernel, documentation is requried to describe what has been changed because it will leave a huge impact not to yourself but other developers.